dev-aws/kafka-shared-msk/billing/billing.tf

```diff
@@ -53,6 +53,24 @@ resource "kafka_topic" "core_bill_rectifications" {
   }
 }
 
+resource "kafka_topic" "billing_engine_events" {
+  name               = "billing.engine-events"
```

Comment:
Im not sure on the name `billing.billing-engine-events` didnt seem that nice?

Reply:
I would go for `billing.billing-engine-events` even though it has some duplication